experanto/interpolators.py

```diff
@@ -546,6 +588,28 @@ def __init__(self, data_file_name, meta_data, cache_data: bool = False) -> None:
         )
 
 
+class EncodedVideoTrial(ScreenTrial):
+    def __init__(self, data_file_name, meta_data, cache_data: bool = False) -> None:
+        super().__init__(
+            data_file_name,
+            meta_data,
+            tuple(meta_data.get("image_size")),
+            meta_data.get("first_frame_idx"),
+            meta_data.get("num_frames"),
+            cache_data=cache_data,
+        )
+        self.video_decoder = VideoDecoder(self.data_file_name)
+
+    def get_data_(self) -> np.array:
+        """Override base implementation to load compressed videos"""
+        full_video = np.transpose(
+            self.video_decoder[:].numpy(), (0, 2, 3, 1)
```

Comment:
is the transpose order specific for Allen data or is it going to the the same for any decoded videos?